satsearch/search.py

```diff
@@ -10,7 +10,7 @@
 
 logger = logging.getLogger(__name__)
 
-API_URL = os.getenv('STAC_API_URL', 'https://1tqdbvsut9.execute-api.us-west-2.amazonaws.com/v0').rstrip('/') + '/'
+API_URL = os.getenv('STAC_API_URL', 'https://earth-search.aws.element84.com/v0').rstrip('/') + '/'
```

Comment:
👋 @matthewhanson what do you think about removing any hardcoded URL? you could still use STAC_API_URL environ but I think it will be better if stac-search wasn't tied to any URL.

We (Devseed) will love to fix the API we are hosting but it means first we have to wait for the specification do be complete.